### PR TITLE
chore: add example for balance AccountStore check (#10167)

### PR DESCRIPTION
**Pull request by ECJ222**
Added an example for balance `AccountStore` 👍

### CODE REVIEW

**Comment by thiolliere** [comment]
fix https://github.com/paritytech/substrate/issues/6532

File: frame/balances/src/lib.rs [hunk 1]
```diff
@@ -222,7 +222,7 @@ pub mod pallet {
 		type ExistentialDeposit: Get<Self::Balance>;
 
 		/// The means of storing the balances of an account.
-		type AccountStore: StoredMap<Self::AccountId, AccountData<Self::Balance>>;
+		type AccountStore: StorageMapShim<Self::Account<Runtime>, frame_system::Provider<Runtime>, AccountId, Self::AccountData<Balance>>;
```
Comment (thiolliere) [inline frame/balances/src/lib.rs:225]: you are not creating an example here, instead you are modifying the library.

An example would look like:
```rust
/// ```
/// construct_runtime!(...)
/// impl Balance for Runtime {... // here you would use storage map shim for the assocaited type account store
/// }
/// ...
/// ```

**Review by thiolliere — CHANGES_REQUESTED** [review]

**Comment by smisty** [comment]
Why are you , whoever you are messing with my self account.  You know my
identity was stolen!

**Comment by smisty** [comment]
I'm am mentally disabled and I don't even know what u r talking about

File: frame/balances/src/lib.rs (hunk 1 above)
Comment (ECJ222) [reply frame/balances/src/lib.rs:225]: Thanks @thiolliere, I will work on it.

**Pull request closed without merging**